utils/calculate_rtf.py

```diff
@@ -20,6 +20,36 @@ def get_parser():
         default=None,
         help="path to logging directory",
     )
+    parser.add_argument(
+        "--log-name",
+        type=str,
+        default="decode",
+        choices=["decode", "asr_inference"],
+        help="name of logfile, e.g., 'decode' (espnet1) and "
+        "'asr_inference' (espnet2)",
+    )
+    parser.add_argument(
+        "--input-shift",
+        type=float,
+        default=10.0,
```

Comment:
It is good to set a default value, but it can be a source of some confusion.
I recommend adding a logging message that this script uses the 10ms shift as default when this option is not specified.

Reply:
Thanks. I will add a logging message that clarifies about the frame shift and default setting.